### ISSUE TITLE
Save sessions as MIDI

### ISSUE THREAD

**Issue by schollz** [schollz/PIanoAI#5]
This simply requires a conversion from the current format (which is a JSON of the notes values and velocities) to the standard MIDI format.

**Comment by kalbhor** [comment]
I'd like to work on this

**Comment by schollz** [comment]
@kalbhor Great! Let me know if you have any questions, I'm happy to help.

**Comment by kalbhor** [comment]
Is there any way to test it without using a keyboard?

**Comment by schollz** [comment]
Yeah. You could test on converting the dumped file format in the [testing/](https://github.com/schollz/PIanoAI/tree/master/testing) directory. The `c_scale.json` should just be a C scale and the `em_jam.json` should be a little song.
